data/deep missions.txt

```diff
@@ -979,13 +980,16 @@ mission "Deep: Project Hawking"
 	description `Fetch research supplies from <stopovers> and return them to <destination> by <date> for "Project Hawking."`
 	source
 		attributes deep
+		attributes outfitter
```

Comment:
Any specific reason for this?

Reply:
Not letting `stations` be the origin of this mission. I don't expect stations to have local markets

Reply:
Fair enough, especially since all Deep worlds have an outfitter but none of the stations do.